libs/wingsdk/src/target-sim/policy.ts

```diff
@@ -20,10 +21,13 @@ export interface PolicyProps {
 
 /**
  * Implementation of `sim.Policy`.
+ *
+ * @inflight `@winglang/sdk.sim.IPolicyClient`
```

Comment:
Missing inflight type marker ?


(Is there a way to enforce this because we have a bunch of places where this is missing)

Reply:
Added some extra rules to the `wing-api-checker`.